resources/default_pipeline_environment.yml

```diff
@@ -164,7 +164,7 @@ steps:
       apiEndpoint: 'https://api.cf.eu10.hana.ondemand.com'
     apiParameters: ''
     loginParameters: ''
-    deployTool: 'cf_native'
+    deployTool: ''
```

Comment:
Why this change?

Reply:
Ah, it is because you want the step to automatically detect the tool. Maybe you could add a comment or even remove it from here. Why an empty string at all?

Reply:
I have added the functionality to set the `deployTool` based on the `buildTool` in case it has not yet been set. If there is the default value, `deployTool` will always be set even if it is not specified.

Reply:
Maybe you could add a comment or even remove it from here. Why an empty string at all?

Reply:
Removed it